spec/git_helper_spec.rb

```diff
@@ -90,4 +91,58 @@
       Fastlane::Helper::GitHelper.commit(message: @message, push: true)
     end
   end
+
+  describe '#is_ignored?' do
+    let(:path) { 'dummy.txt' }
+
+    it 'returns false when the path is not ignored' do
+      setup_git_repo(
+        dummy_file_path: path,
+        in_gitignore: false
+      )
+      expect(Fastlane::Helper::GitHelper.is_ignored?(path: path)).to be false
```

Comment:
I could have used `described_class` here like in the spec above, but I decided not to because the rest of the file doesn't.

(Also, I remembered about `described_class` after writing these tests... 😅 )

Reply:
We still haven't configured CircleCI to run rubocop on each PR (something I really need to take care of before we start reintroducing too many new violations via all our recent updates after my previous cleanup!!) but I know that this would have been flagged by rubocop…

Reply:
🤔 I actually did run Rubocop before submitting my PR (and found some issues, see 79516ac)

What were you expecting it to flag? Right now, it's all green:

(_I tried to upload a screenshot but I'm getting "Something went really wrong" from the editor._ _I also tried to screenshot and upload that..._ 🤦 😅 ) 




I may be able to do it tomorrow / Friday. Were you thinking something simple like "run on every file that changed" or more advanced like comment in line with the violations?

Oh 🤔 Now that I think of it, maybe we should setup Hound?

Reply:
> What were you expecting it to flag? Right now, it's all green:

I was expecting [this rule](https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/DescribedClass) to be triggered 🤔 

> Oh 🤔 Now that I think of it, maybe we should setup Hound?

Woof!! (That means "definitively!!" in case you don't speak dog)